config/initializers/sidekiq.rb

```diff
@@ -1,9 +1,11 @@
+# frozen_string_literal: true
+
 Sidekiq::Client.try(:reliable_push!) unless Rails.env.test?
 
 Sidekiq.configure_server do |config|
   config.try(:reliable!)
 
-  config.redis = System::Application.config.sidekiq
+  config.redis = System::Application.config.sidekiq.merge(namespace: 'system-sidekiq')
```

Comment:
Why not use the template `config/redis.yml`?

Reply:
I guess my first thought was not making it an option, but you are right. I'll change that.